pubnub-gson/pubnub-gson-impl/src/main/java/com/pubnub/internal/java/endpoints/objects_api/members/SetChannelMembersImpl.java

```diff
@@ -123,4 +140,69 @@ public SetChannelMembers uuids(@NotNull final Collection<PNUUID> uuids) {
             };
         }
     }
+
+
+    private List<MemberInput> createMemberInputs() {
+        List<MemberInput> memberInputs = new ArrayList<>();
+        if (users != null) {
+            for (PNUser user: users) {
+                memberInputs.add(new PNMember.Partial(
+                        user.getUserId(),
+                        user.getCustom(), // despite IDE error this works ¯\_(ツ)_/¯
+                        user.getStatus(),
+                        user.getType()
+                ));
+            }
+        } else if (uuids != null) {
+            for (PNUUID uuid : uuids) {
+                memberInputs.add(new PNMember.Partial(
+                        uuid.getUuid().getId(),
+                        (uuid instanceof PNUUID.UUIDWithCustom) ? ((PNUUID.UUIDWithCustom) uuid).getCustom() : null,
+                        uuid.getStatus(),
+                        null
+                ));
+            }
+        }
+        return memberInputs;
+    }
+
+    static MemberInclude getMemberInclude(
+            MemberInclude include,
+            Include.PNUUIDDetailsLevel includeUser,
+            boolean includeTotalCount,
+            boolean includeCustom,
+            boolean includeType
+    ) {
+        if (include != null) {
+            return include;
+        } else {
+            // if deprecated setChannelMembership API use
+            if (includeUser == Include.PNUUIDDetailsLevel.UUID) {
+                return MemberInclude.builder()
+                        .includeTotalCount(includeTotalCount)
+                        .includeCustom(includeCustom)
+                        .includeUserType(includeType)
+                        .includeStatus(true)
+                        .includeUser(true)
+                        .build();
```

Comment:
instead of repeating the builder 3 times, it'll be more concise to have a variable to hold the builder and then

```
if (includeUser == Include.PNUUIDDetailsLevel.UUID) {
    builder.includeUser(true)
}
if (includeUser == Include.PNUUIDDetailsLevel.UUID_WITH_CUSTOM) {
    builder.includeUser(true)
    builder.includeUserCustom(true)
}
return builder.build()
```

Reply:
Good point. I will refactor it.